### DISCUSSION TITLE
Prompt Playground support on phoenix-arize

### DISCUSSION THREAD

**Discussion by mayankkom-dev** [Arize-ai/phoenix#5249] [General]
Hello, Great work with the tool. I am a new user and I am a bit unsure if the prompt Playground feature is available on 5.6.0 release or is it only a part of SaaS - Arize?

I can not see playground button on traces with llm kind.

**Comment by axiomofjoy** [comment]
Hey @mayankkom-dev, prompt playground is our current milestone and will be released in v6. You can follow [this issue](https://github.com/Arize-ai/phoenix/issues/3435) to track progress.

Prompt playground is available in the Arize SaaS platform as well.

Thanks!

**Comment by mikeldking** [comment]
A non streaming version of the prompt playground is available in phoenix SAAS and will be supporting streaming in the coming months! Prompt playground is widely available in the latest phoenix versions. See https://docs.arize.com/phoenix/prompt-engineering/quickstart-prompts
